SocketRocket/Internal/Security/SRPinningSecurityPolicy.h

```diff
@@ -13,6 +13,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// NOTE: While publicly, SocketRocket does not support configuring the security policy with pinned certificates,
```

Comment:
It is a private class as of right now, I don't think full documentation is required at this time. I will change to using /** comments though.